source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -0,0 +1,89 @@
+---
+id: media-middleware
+title: Optimize your media
+---
+
+One of the tough things to do when doing responsive websites is to have images that match your user's screen. Especially when content is contributed by a wide range of people that aren't fully aware of the impact their actions can have on the user's experience.
```

Comment:
I’d add a first short paragraph summarizing the problem.

Suggestions:

> Heavy, unoptimized images make your site noticeably slow for your users and damage their experience.

Also:

```suggestion
One of the tough things to do when doing responsive websites is to have images that match your user's screen. This is especially true when content is contributed by a wide range of people who aren't fully aware of the impact their actions can have on the user's experience. Iterating fast and last-minute design changes may also lead to introducing heavy and unoptimized images without paying attention.
```